menu/additional_apps.sh

```diff
@@ -34,6 +33,11 @@ choice=$(whiptail --title "Which apps do you want to install?" --checklist "Auto
 "SMB-mount" "(Connect to SMB-shares from your local network)" OFF 3>&1 1>&2 2>&3)
 
 case "$choice" in
+    *"Bitwarden "*)
+        clear
+        print_text_in_color "$ICyan" "Downloading Bitwarden_Menu.sh..."
```

Comment:
Actually, I think we should change the info here. It would be better if it said "Downloading script for Bitwarden..."

Reply:
Done